WarOfEmpires.Models/Markets/MarketModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace WarOfEmpires.Models.Markets {
+    public sealed class MarketModel {
+        [DisplayName("Food")]
+        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Food must be a valid number")]
+        public string Food { get; set; }
+        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Food price must be a valid number")]
+        public string FoodPrice { get; set; }
+        [DisplayName("Wood")]
+        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Wood must be a valid number")]
+        public string Wood { get; set; }
+        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Wood price must be a valid number")]
+        public string WoodPrice { get; set; }
+        [DisplayName("Stone")]
+        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Stone must be a valid number")]
+        public string Stone { get; set; }
+        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Stone price must be a valid number")]
+        public string StonePrice { get; set; }
+        [DisplayName("Ore")]
+        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Ore must be a valid number")]
```

Comment:
- Suggest changing the regular expressions for the resources (not the resource prices) to "^\\d{0,8}$" to allow larger quantities to be sold. 

- Suggest changing the validation message for the resources to include a reference to the maximum amount allowed, e.g. "Food must be a valid number and < 100 million" to make it more clear to the user why their transaction was refused

Reply:
I've increased the max to 8 digits but not changed the message for now